File: base/src/main/java/com/windscribe/vpn/api/response/QueryType.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.vpn.api.response;

import androidx.annotation.Keep;

@Keep
public enum QueryType {
    Account(1),
    Technical(2),
    Sales(3),
    Feedback(4);

    private final int value;

    QueryType(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }

    @Override
    public String toString() {
        return name();
    }
}

File: base/src/main/java/com/windscribe/vpn/api/response/QueryType.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.vpn.api.response

import androidx.annotation.Keep

@Keep
enum class QueryType(
    val value: Int,
) {
    Account(1),
    Technical(2),
    Sales(3),
    Feedback(4),
    ;

    override fun toString(): String = name
}
